src/jdk.jdwp.agent/share/native/libjdwp/util.h

```diff
@@ -87,6 +87,7 @@ typedef struct {
     jboolean vthreadsSupported; /* If true, debugging support for vthreads is enabled. */
     jboolean includeVThreads;   /* If true, VM.AllThreads includes vthreads. */
     jboolean rememberVThreadsWhenDisconnected;
+    jboolean virtualThreadStartEventsPermanentlyEnabled;
```

Comment:
Could you please add description for this 2 flags (and maybe `vthreadsSupported` description also can be enhanced to see relations between the flags)